gomint-server/src/main/java/io/gomint/server/command/gomint/PluginsCommand.java

```diff
@@ -26,6 +26,19 @@
     @Override
     public CommandOutput execute(CommandSender commandSender, String alias, Map<String, Object> arguments) {
         Map<String, Plugin> plugins = GoMint.instance().getPluginManager().getPlugins();
-        return CommandOutput.successful(String.join(", ", plugins.keySet()));
+
+        if (plugins.isEmpty()) {
+            return CommandOutput.failure("No plugins were loaded.");
+        }
+
+        StringBuilder stringBuilder = new StringBuilder("Plugins (§a" + plugins.size() + "§r): §a");
+
+        for (Plugin plugin : plugins.values()) {
+            stringBuilder.append(plugin.getName()).append("§r, ");
+        }
+
+        stringBuilder.setLength(stringBuilder.length() - 2);
```

Comment:
```suggestion
        stringBuilder.setLength(stringBuilder.length() - 4);
```

Reply:
To get rid of the last §r?

Reply:
Yes, its not making a visual difference, but we don't need it.